storage/statedb/cache_hybrid.go

```diff
@@ -54,3 +54,17 @@ func (cache *hybridCache) Has(k []byte) ([]byte, bool) {
 	}
 	return cache.remote.Has(k)
 }
+
+func (cache *hybridCache) SaveToFile(filePath string, concurrency int) error {
+	if err := cache.local.SaveToFile(filePath, concurrency); err != nil {
+		logger.Error("failed to save local cache to file",
+			"filePath", filePath, "concurrency", concurrency, "err", err)
+		return err
+	}
+	if err := cache.remote.SaveToFile(filePath, concurrency); err != nil {
```

Comment:
I think we don't need to save remote cache dump to local.

Reply:
Currently this does nothing but just calling the empty function.
As I added SaveToFile to the `Cache` interface, I also write the lines to call `SaveToFile` just in case.

Reply:
I know it is an empty function. 
But if you want to keep the interface, do we need to set different file path?

Reply:
Ok, I understood your point. After the test, let's remove the remote part.